### ISSUE TITLE
AuralGrapheme - problem with certain time values

### ISSUE THREAD

**Issue by Sciss** [Sciss/SoundProcesses#35]
If we change the grid here

https://github.com/Sciss/Mellite-website/blob/09bebe904de559ab2078f2f219343832602e129a/snippets/src/main/scala/de/sciss/soundprocesses/tutorial/Snippet6.scala#L52

From 0.511 seconds to 0.5 seconds per beat, we miss most of the pitches; it seems there is a bug with the regular look-ahead-pulse coinciding with the grapheme's time values.

**Comment by Sciss** [comment]
A log showing how `playView` is somehow skipped, probably as interference with `gridReached`:

```
[5 Sep 2017, 22:50'30.408] proc - aural Started server
[5 Sep 2017, 22:50'30.444] proc - aural create context 231748bc
[5 Sep 2017, 22:50'30.513] proc - aural try build Proc<2> - <unnamed>
[5 Sep 2017, 22:50'30.550] proc - aural acceptedInputs += AttributeKey(pitch) -> Input.Scalar.Value(numChannels = 1)
[5 Sep 2017, 22:50'30.605] proc - aural buildAdvanced Proc<2>; complete? true
[5 Sep 2017, 22:50'30.612] proc - aural begin prepare Proc<2> (7e6c0d9d)
[5 Sep 2017, 22:50'30.615] proc - aural begin launch  Proc<2> (7e6c0d9d)
[5 Sep 2017, 22:50'30.673] proc - aural scheduled - prepare IntAttribute(pitch)@17fa114f - TimeRef(span = Span(0,7056000) / (0.000s - 0.500s), frame = 0 / 0.000s)
[5 Sep 2017, 22:50'30.678] proc - aural scheduled - prepare IntAttribute(pitch)@7d0f0d42 - TimeRef(span = Span(7056000,14112000) / (0.500s - 1.000s), frame = 0 / 0.000s)
[5 Sep 2017, 22:50'30.679] proc - aural scheduled - prepare IntAttribute(pitch)@1cd80063 - TimeRef(span = Span(14112000,21168000) / (1.000s - 1.500s), frame = 0 / 0.000s)
[5 Sep 2017, 22:50'30.682] proc - aural grapheme - playView: IntAttribute(pitch)@17fa114f - TimeRef(span = Span(0,7056000) / (0.000s - 0.500s), frame = 0 / 0.000s)
[5 Sep 2017, 22:50'30.687] proc - aural scheduled - scheduleNextEvent(0 / 0.000s) -> 7056000 / 0.500s
[5 Sep 2017, 22:50'30.715] proc - aural scheduled - scheduleGrid(0 / 0.000s, 21168000 / 1.500s) -> 7056000 / 0.500s
[5 Sep 2017, 22:50'30.721] proc - aural playOutputs Proc<2>
[5 Sep 2017, 22:50'30.721] proc - aural launched Proc<2> -> AuralProc(Synth(id=1000, def=proc_0)) (7e6c0d9d)
not0: 78
[5 Sep 2017, 22:50'31.160] proc - aural scheduled - eventReached(7056000 / 0.500s)
[5 Sep 2017, 22:50'31.161] proc - aural grapheme - playView: IntAttribute(pitch)@7d0f0d42 - TimeRef(span = Span(7056000,14112000) / (0.500s - 1.000s), frame = 7056000 / 0.500s)
[5 Sep 2017, 22:50'31.162] proc - aural aural - stopView: IntAttribute(pitch)@17fa114f
[5 Sep 2017, 22:50'31.166] proc - aural scheduled - scheduleNextEvent(7056000 / 0.500s) -> 14112000 / 1.000s
[5 Sep 2017, 22:50'31.167] proc - aural scheduled - gridReached(7056000 / 0.500s)
[5 Sep 2017, 22:50'31.169] proc - aural scheduled - prepare IntAttribute(pitch)@1417d7cb - TimeRef(span = Span(21168000,28224000) / (1.500s - 2.000s), frame = 7056000 / 0.500s)
[5 Sep 2017, 22:50'31.170] proc - aural scheduled - scheduleGrid(7056000 / 0.500s, 28224000 / 2.000s) -> 14112000 / 1.000s
[5 Sep 2017, 22:50'31.171] proc - aural ...reschedule
[5 Sep 2017, 22:50'31.171] proc - aural scheduled - scheduleNextEvent(7056000 / 0.500s) -> 14112000 / 1.000s
note: 79
[5 Sep 2017, 22:50'31.608] proc - aural scheduled - gridReached(14112000 / 1.000s)
[5 Sep 2017, 22:50'31.611] proc - aural scheduled - prepare IntAttribute(pitch)@2aedb8c8 - TimeRef(span = Span(28224000,35280000) / (2.000s - 2.500s), frame = 14112000 / 1.000s)
[5 Sep 2017, 22:50'31.613] proc - aural scheduled - scheduleGrid(14112000 / 1.000s, 35280000 / 2.500s) -> 21168000 / 1.500s
[5 Sep 2017, 22:50'31.614] proc - aural ...reschedule
[5 Sep 2017, 22:50'31.615] proc - aural scheduled - scheduleNextEvent(14112000 / 1.000s) -> 21168000 / 1.500s
[5 Sep 2017, 22:50'32.108] proc - aural scheduled - gridReached(21168000 / 1.500s)
[5 Sep 2017, 22:50'32.111] proc - aural scheduled - prepare IntAttribute(pitch)@4623dad6 - TimeRef(span = Span(35280000,42336000) / (2.500s - 3.000s), frame = 21168000 / 1.500s)
[5 Sep 2017, 22:50'32.113] proc - aural scheduled - scheduleGrid(21168000 / 1.500s, 42336000 / 3.000s) -> 28224000 / 2.000s
[5 Sep 2017, 22:50'32.114] proc - aural ...reschedule
[5 Sep 2017, 22:50'32.115] proc - aural scheduled - scheduleNextEvent(21168000 / 1.500s) -> 28224000 / 2.000s
[5 Sep 2017, 22:50'32.608] proc - aural scheduled - gridReached(28224000 / 2.000s)
[5 Sep 2017, 22:50'32.610] proc - aural scheduled - prepare IntAttribute(pitch)@70059927 - TimeRef(span = Span(42336000,49392000) / (3.000s - 3.500s), frame = 28224000 / 2.000s)
[5 Sep 2017, 22:50'32.612] proc - aural scheduled - scheduleGrid(28224000 / 2.000s, 49392000 / 3.500s) -> 35280000 / 2.500s
[5 Sep 2017, 22:50'32.614] proc - aural ...reschedule
[5 Sep 2017, 22:50'32.614] proc - aural scheduled - scheduleNextEvent(28224000 / 2.000s) -> 35280000 / 2.500s
[5 Sep 2017, 22:50'33.108] proc - aural scheduled - gridReached(35280000 / 2.500s)
[5 Sep 2017, 22:50'33.110] proc - aural scheduled - prepare IntAttribute(pitch)@4256702e - TimeRef(span = Span(49392000,56448000) / (3.500s - 4.000s), frame = 35280000 / 2.500s)
[5 Sep 2017, 22:50'33.112] proc - aural scheduled - scheduleGrid(35280000 / 2.500s, 56448000 / 4.000s) -> 42336000 / 3.000s
[5 Sep 2017, 22:50'33.114] proc - aural ...reschedule
[5 Sep 2017, 22:50'33.115] proc - aural scheduled - scheduleNextEvent(35280000 / 2.500s) -> 42336000 / 3.000s
[5 Sep 2017, 22:50'33.608] proc - aural scheduled - gridReached(42336000 / 3.000s)
[5 Sep 2017, 22:50'33.610] proc - aural scheduled - prepare IntAttribute(pitch)@266d954f - TimeRef(span = From(56448000) / (4.000s - inf), frame = 42336000 / 3.000s)
[5 Sep 2017, 22:50'33.612] proc - aural ...reschedule
[5 Sep 2017, 22:50'33.612] proc - aural scheduled - scheduleNextEvent(42336000 / 3.000s) -> 49392000 / 3.500s
[5 Sep 2017, 22:50'34.108] proc - aural scheduled - eventReached(49392000 / 3.500s)
[5 Sep 2017, 22:50'34.109] proc - aural grapheme - playView: IntAttribute(pitch)@4256702e - TimeRef(span = Span(49392000,56448000) / (3.500s - 4.000s), frame = 49392000 / 3.500s)
[5 Sep 2017, 22:50'34.110] proc - aural aural - stopView: IntAttribute(pitch)@7d0f0d42
[5 Sep 2017, 22:50'34.119] proc - aural scheduled - scheduleNextEvent(49392000 / 3.500s) -> 56448000 / 4.000s
note: 85
[5 Sep 2017, 22:50'34.609] proc - aural scheduled - eventReached(56448000 / 4.000s)
[5 Sep 2017, 22:50'34.611] proc - aural grapheme - playView: IntAttribute(pitch)@266d954f - TimeRef(span = From(56448000) / (4.000s - inf), frame = 56448000 / 4.000s)
[5 Sep 2017, 22:50'34.612] proc - aural aural - stopView: IntAttribute(pitch)@4256702e
note: 86
[5 Sep 2017, 23:04'54.369] proc - aural Stopped server
```

**Comment by Sciss** [comment]
fixed in 56d46c2432d9d5571de83923ffdbc36ca6737115
